### ISSUE TITLE
File handling snippet in any language.

### ISSUE THREAD

**Issue by Anuja-19** [Anuja-19/Snippets#2]
Write a snippet to create, read and write a file in any language.

**Comment by Utsav-Ladani** [comment]
Hey @Anuja-19 , I want to add snippet of file handling in C++. Can I add ?

**Comment by GanapathiSubramanian** [comment]
Hey @Anuja-19 , Can I do this in JavaScript?

**Comment by Anuja-19** [comment]
Yes you can add in any language @GanapathiSubramanian  @Utsav-Ladani

**Comment by Utsav-Ladani** [comment]
Thanks @Anuja-19 :)

**Comment by namhsuya** [comment]
Hello, @Anuja-19!
I have added a filehandling directory, and under python I've added the snippets for file handling in python.
Hope this help.
Cheers~

**Comment by Anuja-19** [comment]
Yes, Kudos!! @namhsuya

**Comment by Utsav-Ladani** [comment]
Hey @Anuja-19 , please review and merge PR.
Thanks :)

**Comment by vieee** [comment]
Hey @Anuja-19, I want to add snippet of file handling in Java. Can I add?

**Comment by Anuja-19** [comment]
Sure, @vieee , assigning it to you

**Comment by Ephiret** [comment]
@Anuja-19 , I would like to add snippet of file handling in python. Can you assign me this issue?

**Comment by namhsuya** [comment]
> @Anuja-19 , I would like to add snippet of file handling in python. Can you assign me this issue?

@Ephiret I believe I have already added a python version. @Anuja-19 , if you could please confirm? 
Cheers~

**Comment by Anuja-19** [comment]
> > @Anuja-19 , I would like to add snippet of file handling in python. Can you assign me this issue?
> 
> @Ephiret I believe I have already added a python version. @Anuja-19 , if you could please confirm?
> Cheers~

Yes, it is already done. Please work on some other issue

**Comment by ishitacodes** [comment]
Hey @Anuja-19, I want to add snippet of file handling in Php. Can I add?

**Comment by ChiefMilesEdgeworth** [comment]
I'd like to do this in Rust. Can you add me?
